src/table/ResourceTable.tsx

```diff
@@ -33,27 +33,27 @@ export class ResourceTable extends React.Component<
   ResourceTableState
 > {
   state: ResourceTableState = {
+    sortedInfo: undefined,
     selectedRowKeys: []
   };
 
   handleChange = (
     pagination: PaginationConfig,
     filters: Record<any, string[]>,
-    sorter: SorterResult<any>
+    sorter: SortInfo
```

Comment:
Please keep the `SorterResult<any>` instead of SortInfo, it's antd type and should refer to this method used in antd table property